planemo/commands/cmd_workflow_edit.py

```diff
@@ -18,14 +18,14 @@
 def cli(ctx, workflow_identifier, output=None, force=False, **kwds):
     """Open a synchronized Galaxy workflow editor."""
     assert is_galaxy_engine(**kwds)
-    runnable = for_runnable_identifier(ctx, workflow_identifier, kwds.get("profile"))
+    runnable = for_runnable_identifier(ctx, workflow_identifier, kwds)
 
     kwds["workflows_from_path"] = True
 
     with engine_context(ctx, **kwds) as galaxy_engine:
         with galaxy_engine.ensure_runnables_served([runnable]) as config:
             workflow_id = config.workflow_id_for_runnable(runnable)
-            url = f"{config.galaxy_url}/workflow/editor?id={workflow_id}"
+            url = f"{config.galaxy_url}/workflows/edit?id={workflow_id}"
```

Comment:
Does this break with old Galaxy releases?

Reply:
Yes, but I'm gonna say that change has happened about 2 years ago ? And it's been broken since, so my thinking is it's not worth making that conditional on the version ?

Reply:
Fine by me!